api/common.js

```diff
@@ -419,10 +419,10 @@ exports.load_github_detail = function(service_name, cb) {
 }
 
 exports.generateQuery = function(str) {
-    let result=stopwords.cleanText(str).split(' ').filter(e => String(e).trim());
+    let result=stopwords.cleanText(str.replaceAll(/[^a-zA-Z ]/g,"")).split(' ').filter(e => String(e).trim());
     if(result.length == 0) return null;
     result = result.map((word) => `W='${word}'`).join(',');
-    return "OR("+result+")"
+    return "And(OR("+result+"),Composite(F.FN=='neuroscience'))"
```

Comment:
I think this makes the papers more relevant to neuroscience, but won't it make it impossible to find paper that are not in neuroscience but might be related cross-diciplinary? Is there a way to put *weight* toward neuroscience without completely filtering out non-neuroscience papers?

Reply:
A paper has multiple fields this will only filter papers that have one of the fields as neuroscience and the rest can be any

Reply:
You are using `And()` logic there, so if FN is not 'neuroscience`, then paper won't show up, even if rest of the fields looks promising, right?

Reply:
yes !

Reply:
however, it does show cross-disciplinary paper

Reply:
instead of looking for keywords in the title how about, we look in the abstract and remove the field of neuroscience?